draft-ietf-quic-ack-frequency.md

```diff
@@ -342,20 +342,18 @@ see {{out-of-order}}.
 
 It is important to receive timely feedback after long idle periods,
 e.g. update stale RTT measurements. When no acknowledgment has been sent in
-over one smoothed round trip time, receivers are encouraged to send an
-acknowledgment soon after receiving an ack-eliciting packet. This is not an
-issue specific to this document, but the mechanisms specified herein could
-create excessive delays.
+over one smoothed round trip time ({{Section 5.3 of QUIC-RECOVERY}}),
+receivers are encouraged to send an acknowledgment soon after receiving
+an ack-eliciting packet. This is not an issue specific to this document,
+but the mechanisms specified herein could create excessive delays.
 
 ## Response to Out-of-Order Packets {#out-of-order}
 
 As specified in {{Section 13.2.1 of QUIC-TRANSPORT}}, endpoints are expected to
-send an acknowledgment immediately on receipt of a reordered ack-eliciting
-packet with a smaller packet number than the highest-numbered ack-eliciting packet
-or with a higher packet number when there are missing packets between that packet
-and the highest-numbered ack-eliciting packet.
-This extension modifies that behavior when an ACK_FREQUENCY frame with
-a Reordering Threshold value other than 1 has been received.
+send an immediate acknowledgment upon receipt of a reordered ack-eliciting
+packet. After an ACK_FREQUENCY frame with a Reordering Threshold value other
```

Comment:
Not sure that makes a real difference but maybe "if" is better here:
```suggestion
packet. If an ACK_FREQUENCY frame with a Reordering Threshold value other
```

Reply:
I slightly prefer After, but either works